example/lib/movas/observables/app_config_observable.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:movas_example/movas/services/posts/posts_service.dart';
+
+class AppConfigO {
+  AppConfigO(
```

Comment:
never use service in observable.

observables only hold imutable data or classes. not business logic